bundler.d/example.rb

```diff
@@ -6,6 +6,8 @@
 # or `ensure_gem` to make sure a gem is there w/o worrying about if it is an
 # override or not
 
-ensure_gem "sentry-ruby"
-ensure_gem "sentry-rails"
+gem "sentry-ruby"
```

Comment:
Since removing form the gemspec needing it to be gem bc we are adding to hyku form the knapsack via inject-bundler